### DISCUSSION TITLE
Combine flux bootstrap and flux install

### DISCUSSION THREAD

**Discussion by Thaval** [fluxcd/flux2#3985] [Proposals]
Hi, 

I struggle a little with an easy setup and managing the yamls. 
So `flux boostrap` does a great job setting up everything, but it doesnt give me the `gotk-components` files like `flux install --export`. 
Is it always safe to run `flux install --export` for the previous `flux bootstrap` command? 
I think we should have an option for the bootstrap command will just act like `flux install --export`, so giving us the manifests.

Example:
```
flux bootstrap git `
 --export
 --path="clusters/my-cluster/flux-system"
```
This will install flux-system and additionally export the manifests.

**Comment by stefanprodan** [comment]
> Is it always safe to run flux install --export for the previous flux bootstrap command?

Yes, bootstrap executes the same code as install with the Git operations on top. Internally we have a Go package for generating the manifests, this code is shared between flux install, flux bootstrap and the Flux Terraform provider.

**Reply by Thaval** [reply]
Do I need a RC relese for that to happen? Im missing the manifest files (gotk-components, gotk-sync and kustomization.yaml) but it seems to be pushing to git.
